src/sqs_adaptor.py

```diff
@@ -57,6 +57,10 @@ def __init__(self, queue_name):
 
     def write(self, string):
         "called when given a validated response"
+        # totally naive. unicode check probably needed
+        if len(string) >= 256000:
```

Comment:
Reading https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-limits.html#w117aac20b9b5 it's actually KiB, so 262144 (though I suspect the missing 6144 bytes won't be that useful).